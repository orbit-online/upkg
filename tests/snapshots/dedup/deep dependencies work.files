[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .packages
│   │   ├── [drwxr-xr-x]  dep-1@51511148e1748167fbb0e4d5f0d46f8b45d06fe6cbb22d314e72d97cdbb2c4db
│   │   │   ├── [drwxr-xr-x]  .upkg
│   │   │   │   ├── [lrwxrwxrwx]  .packages -> ../../
│   │   │   │   └── [lrwxrwxrwx]  dep-2 -> .packages/dep-2@19a270455443770ee8db2e2f3b1b321bd48e1be756184ad80e6af796eade871e
│   │   │   └── [-rw-r--r--]  upkg.json
│   │   ├── [drwxr-xr-x]  dep-2@19a270455443770ee8db2e2f3b1b321bd48e1be756184ad80e6af796eade871e
│   │   │   ├── [drwxr-xr-x]  .upkg
│   │   │   │   ├── [lrwxrwxrwx]  .packages -> ../../
│   │   │   │   └── [lrwxrwxrwx]  dep-3 -> .packages/dep-3@43ef7ec15c6717ee46331599a33d791c7e6b5448eedcfd2b1391dca26a86bdf4
│   │   │   └── [-rw-r--r--]  upkg.json
│   │   ├── [drwxr-xr-x]  dep-3@43ef7ec15c6717ee46331599a33d791c7e6b5448eedcfd2b1391dca26a86bdf4
│   │   │   ├── [drwxr-xr-x]  .upkg
│   │   │   │   ├── [lrwxrwxrwx]  .packages -> ../../
│   │   │   │   └── [lrwxrwxrwx]  dep-4 -> .packages/dep-4@e59be1b9a797d85bc901502db1a4b9f7b194f17972c7d21ae1206808f32f33db
│   │   │   └── [-rw-r--r--]  upkg.json
│   │   ├── [drwxr-xr-x]  dep-4@e59be1b9a797d85bc901502db1a4b9f7b194f17972c7d21ae1206808f32f33db
│   │   │   ├── [drwxr-xr-x]  .upkg
│   │   │   │   ├── [lrwxrwxrwx]  .packages -> ../../
│   │   │   │   └── [lrwxrwxrwx]  dep-5 -> .packages/dep-5@5a4173f8e0e51b3e5ec8d22e29a2110e84dd4b42dc6a7b56147869410cbe2373
│   │   │   └── [-rw-r--r--]  upkg.json
│   │   └── [drwxr-xr-x]  dep-5@5a4173f8e0e51b3e5ec8d22e29a2110e84dd4b42dc6a7b56147869410cbe2373
│   │       └── [-rw-r--r--]  upkg.json
│   └── [lrwxrwxrwx]  dep-1 -> .packages/dep-1@51511148e1748167fbb0e4d5f0d46f8b45d06fe6cbb22d314e72d97cdbb2c4db
└── [-rw-r--r--]  upkg.json
