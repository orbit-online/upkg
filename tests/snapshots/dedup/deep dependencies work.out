upkg: No checksum given for '$BATS_RUN_TMPDIR/package-fixtures/default/dep-1.tar', determining now
upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/default/dep-1.tar'
